packages/cache/src/index.js

```diff
@@ -42,11 +47,11 @@ let getServerStylisCache = isBrowser
       }
     })
 
+const defaultStylisPlugins = [compat, prefixer]
```

Comment:
Why is compat not an omnipresent plugin?

Reply:
it's up for discussion, main reason is that omnipresent ones come last currently and this probably should come first. I see omnipresent as really essentials and compat is not that - using a custom stylis plugins is already a really advanced use case and it seems that greater control over plugin ordering is good, by adding `compat` manually we have to decide about its order.

Reply:
Ok, I've pushed around some bits and here you go: https://github.com/emotion-js/emotion/pull/1817/commits/48629c723c59d6203694875c3760c8cfab2cc65f